docker-compose.yml

```diff
@@ -249,23 +249,30 @@ services:
 
   ##flink
   jobmanager:
+    build:
+      context: pinpoint-flink
+      dockerfile: Dockerfile
     container_name: "${PINPOINT_FLINK_NAME}-jobmanager"
-    image: flink:1.3.1
+    image: ${PINPOINT_FLINK_NAME}-flink:dev
```

Comment:
image: here is just to tag the build done by docker compose, so flink version will be taken from the Dockerfile. It's just to avoid bulding twice the image for jobmanager and taskmanager. We cant set it to 1.7 to avoid thinking it's another version.